generation/core/generator.js

```diff
@@ -215,7 +243,7 @@ module.exports = function({
 			: typeCheckCreator(json);
 	}
 
-	return function(files) {
+	return function (files) {
```

Comment:
Again, javascript voodoo. I'm too old for this magic. ;(

Reply:
What is going on here?